js/data/defaultSearchProviders.js

```diff
@@ -0,0 +1,26 @@
+const defaultSearchProviders = [
+  {
+    id: 'ob1',
+    name: 'OB1',
+    logo: '../imgs/ob1searchLogo.png',
+    search: 'https://search.ob1.io/search',
+    listings: 'https://search.ob1.io/search/listings',
+    torsearch: 'http://obsearchwhcqnhvj.onion/search',
+    torlistings: 'http://obsearchwhcqnhvj.onion/search/listings',
+    locked: true,
+    order: 1,
```

Comment:
Is this order attribute really necessary? If the only reason for it is to ensure the default ones are first, can we just splice them in at the beginning?

`app.searchProviders.add(getDefaultSearchProviders(), { at: 0 });`

Reply:
The idea here is to make sure no other process can accidentally break things by putting other models in front of them.

I'll go change the code that assumes the original default provider is at position 0, that's a fragile way of doing it.